flow/connectors/mysql/mysql.go

```diff
@@ -182,6 +182,10 @@ func (c *MySqlConnector) connect(ctx context.Context) (*client.Conn, error) {
 		if _, err := conn.Execute("SET sql_mode = 'ANSI,NO_BACKSLASH_ESCAPES'"); err != nil {
 			return nil, fmt.Errorf("failed to set sql_mode to ANSI: %w", err)
 		}
+		// set max_execution_time to unlimited
+		if _, err := conn.Execute("SET SESSION MAX_EXECUTION_TIME=0;"); err != nil {
```

Comment:
another one of those things where MySQL and Maria differ, they call it `max_statement_time`
Also older MySQL does not have this: https://github.com/sixhop/AutoMySQLBackup/issues/31